athenz-sia.env

```diff
@@ -38,13 +38,18 @@ DNS_SUFFIX=athenz.cloud
 #     "DNS: <Instance ID>.instanceid.athenz.<Provider DNS Suffix>"
 #   However, in some cases we need to have extra pattern to use the certificates for TLS termination.
 #     "DNS: <any prefix name>.<Provider DNS Suffix>"
-#   This field adds extra value in SANs DNS filed to cover such senarios.
+#   This field adds extra value in SANs DNS filed to cover such scenarios.
 # For example: CERT_EXTRA_SAN_DNS=strings.that.does.not.match.with.the.athenz.domain.name.athenz.cloud
 # Note: <Provider DNS Suffix> must meat the suffix registered in Athenz (e.g. ".athenz.cloud")
 # Default value for binary: https://github.com/AthenZ/k8s-athenz-sia/blob/c8478297a9d228ffc0a6a1ea469ad0ef8a682dc8/pkg/config/default.go#L42
 #
 CERT_EXTRA_SAN_DNS=
 #
+# X.509 certificate subject field in RFC 4514 format, e.g. C=JP,ST=Tokyo,O=Example Corporation
+# Setting CN, SERIALNUMBER and other unknown attributes are not allowed. OU attribute will only apply to role certificate, as in Athenz specification, OU=PROVIDER_SERVICE for service certificate.
```

Comment:
write it in bullet points please.

Reply:
fixed in [383b6c3](https://github.com/AthenZ/k8s-athenz-sia/pull/185/commits/383b6c3f42b977f38baea6fbc11b96c9ebd60179)